src/components/FilterBar/FilterDisplay/Inputs/RelativeDateRangeInput.react.js

```diff
@@ -201,8 +257,8 @@ function relativeOptions() {
       from: moment().startOf('isoWeek').diff(moment(), 'days'),
       to: moment().endOf('isoWeek').diff(moment(), 'days')
     },
-    'Older than 7 days': { from: null, to: -8 },
-    'Older than 14 days': { from: null, to: -15 },
-    'Older than 20 days': { from: null, to: -21 },
+    'Older than 7 days': { from: -8, to: 0 },
+    'Older than 14 days': { from: -15, to: 0 },
+    'Older than 20 days': { from: -21, to: 0 },
```

Comment:
Does this work?
I would think that the previous values for this would have been correct.
"null to -8" gives us the date range of "forever ago to 8 days ago" (so, "Older than 7 days"), right?
"-8 to 0" seems to be "Within the last 7 days" to me. Am I missing a logic change here?

Same for the two changes below.

Reply:
@SyntheticDave I am a little unsure about that. The purpose of this filter is to create a range always based on today. The option "null to -8" creates an impossible range since null is translated to today, so how can we end up with "from" that starts after "to"?